### ISSUE TITLE
[Free] Make the default tags actionable on creating a new import

### ISSUE THREAD

**Issue by vytisbulkevicius** [Codeinwp/feedzy-rss-feeds#1002]
### Description

When creating a new import tags added by default are without actions, this is confusing. We should add tags the same way as they are chosen from the dropdown.

Currently it looks like this:
![Image](https://github.com/user-attachments/assets/ca5edbca-367c-49d8-9842-29d294b5967c)

Should be:
![Image](https://github.com/user-attachments/assets/4aca3c2c-1ab8-498d-afa3-9c0874a13cbf)

### Step-by-step reproduction instructions

-

### Is the issue you are reporting a regression

No

**Comment by HardeepAsrani** [comment]
@vytisbulkevicius I get the idea here but I'm not 100% sure what happens when you click on the icon on the right of the items?

**Comment by selul** [comment]
@HardeepAsrani the default import created from wizard is not having the tags properly parsed.

**Comment by HardeepAsrani** [comment]
@selul Yes, I had noticed it after this PR was merged and had updated that part here: https://github.com/Codeinwp/feedzy-rss-feeds/pull/1035
